feature/policy_forwarding/encapsulation/otg_tests/decap_gre_ipv4/README.md

```diff
@@ -36,14 +36,21 @@ The test also confirms the correct forwarding of traffic not matching the decaps
     *  Configure static routes for destination IPV4-DST2 and IPV6-DST2 towards ATE Port 2.
 
 3.  Policy-Based Forwarding: 
-    *  Rule 1: Match GRE traffic with destination DECAP-DST and decapsulate.
+    *  Rule 1: Match GRE traffic with destination DECAP-DST using destination-address-prefix-set and decapsulate.
     *  Rule 2: Match all other traffic and forward (no decapsulation).
-    *  Apply a policy with the following rules to the ingress interface. 
+    *  Apply the defined policy with to the ingress ATE Port 1 interface. 
     
-    **TODO:** OC model does not have provision to apply decap policy to network-instance level does not exist. Needs clarification and/or definition by vendors.
+    **TODO:** OC model does not have a provision to apply decap policy at the network-instance level for traffic destined to device loopback interface (see Cisco CLI config exepmt below). Needs clarification and/or augmentation by vendors if required.
```

Comment:
Add reference to pull request in progress:
https://github.com/openconfig/public/pull/1150